spring-cloud-gcp-pubsub/src/test/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapterTests.java

```diff
@@ -49,45 +45,41 @@
  * @author João André Martins
  * @author Doug Hoard
  */
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
OK. Each of us has its own codding style and vision how it must look in the end.
I can your PR since it looks like we are not going to agree in some compromise, but I would like to quote JavaDoc of this class:
```
* Using this implementation of the runner is not recommended.
* Engineers should care for removing unused stubbings because they are dead code,
* they add unnecessary details, potentially making the test code harder to comprehend.
* If you have good reasons to use the silent runner, let us know at the mailing list
* or raise an issue in our issue tracker.
* The purpose of silent implementation is to satisfy edge/unanticipated use cases,
* and to offer users an opt-out.
```

So, did you find the real reason to use it in your tests? Have you contacted with Mockito team about your reason? 😉

Reply:
I changed to use `MockitoJUnitRunner.Silent.class` because I wanted to set the mock for `nack` even though it shouldn't be called.  A call to `nack` would result in the `value` being set, and hence an error.

With the refactored code, I have reverted this change.